operators/test/e2e/test/apmserver/http_client.go

```diff
@@ -40,11 +42,22 @@ func NewApmServerClient(as apmtype.ApmServer, k *test.K8sClient) (*ApmClient, er
 		return nil, err
 	}
 
+	scheme := "http"
+	var caCerts []*x509.Certificate
+	if as.Spec.HTTP.TLS.Enabled() {
+		scheme = "https"
+		crts, err := k.GetHTTPCerts(name.APMNamer, as.Name)
+		if err != nil {
+			return nil, err
+		}
+		caCerts = crts
+	}
+
 	inClusterURL := fmt.Sprintf(
-		"http://%s.%s.svc:%d", as.Status.ExternalService, as.Namespace, config.DefaultHTTPPort,
+		"%s://%s.%s.svc:%d", scheme, as.Status.ExternalService, as.Namespace, config.DefaultHTTPPort,
```

Comment:
Unrelated to your PR, but maybe we should remove the `.svc` bit, similar to what @david-kow did in https://github.com/elastic/cloud-on-k8s/pull/1243/files?

Reply:
This works now because the SANs have been fixed, right?

Reply:
Hm, changing this seems to break the ability to run e2e locally